### DISCUSSION TITLE
Control how line breaks and hyphens are used

### DISCUSSION THREAD

**Discussion by johnmathews** [tailwindlabs/tailwindcss#3985] [Help]
I use tailwind on my blog and and on small screens (with article titles that have a large font size) I see that tailwind or maybe just the underlying css likes to have at least 2 words on a line before starting a new line. If 2 words will not fit then the second word is hyphenated so that there is never just one word per line. This is annoying.

I would like to avoid hyphenation and use line breaks wherever possible, in a particular element class. I would prefer to see one word per line, rather than hyphenated words.

I do not know how to force this or change the criteria used to decide between hyphenation and line breaks.

Is this possible?

I'm aware of css `word-break` options but these do not have the desired effect. 

Thanks for any thoughts or ideas!

**Comment by fratzinger** [comment]
I've created a PR: https://github.com/tailwindlabs/tailwindcss/pull/10071
